docs/publishing/publishing.md

```diff
@@ -1,3 +1,7 @@
+---
+description: You can instantly publish and newly created Content item or save its draft for later editing. Drafts are autosaved to prevent you from losing your work.
```

Comment:
```suggestion
description: You can instantly publish a newly created Content item or save its draft for editing. Drafts are autosaved to prevent you from losing your work.
```